ami-recycle-bin/example-pattern.json

```diff
@@ -0,0 +1,55 @@
+{
+    "title": "Deregister expired AMIs and its snapshot with AWS Lambda and retain them in Amazon EC2 Recycle Bin",
+    "description": "This project demonstrates a pattern to deregister and retain expired AMI and its snapshot with AWS Lambda and Amazon EC2 Recycle Bin",
+    "language": "YAML",
```

Comment:
You can leave this property empty for Terraform

Reply:
> You can leave this property empty for Terraform

Updated the `language` property value to be an empty string